test/msw-api/setup-server/scenarios/custom-transformers.test.ts

```diff
@@ -11,7 +11,7 @@ const jsonBig = (body: Record<string, any>): ResponseTransformer => {
 }
 
 const server = setupServer(
-  rest.get('http://test.mswjs.io/me', (req, res, ctx) => {
+  rest.get('http://test.mswjs.io/me', (req, res, _ctx) => {
```

Comment:
I think we should drop the unused arguments if they are the last in the arguments list. 

```diff
-(req, res, _ctx)  => {
+(req, res) => {
```

Reply:
Sure. I like the dropping approach.